pkg/kubelet/hyper/hyper.go

```diff
@@ -812,18 +885,12 @@ func (r *runtime) RunPod(pod *api.Pod, restartCount int, pullSecrets []api.Secre
 	}
 
 	// Create and start hyper pod
-	podSpec, err := r.getPodSpec(podFullName)
```

Comment:
This is how kubestack re-write that `podSpec` file: https://github.com/hyperhq/kubestack/blob/master/pkg/plugins/openvswitch/openvswitch.go#L257